packages/opentelemetry-core/src/platform/browser/log-level.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { LogLevel } from '../../common/types';
+
+/**
+ * @returns default log level until below issue is tackled.
+ * TODO: https://github.com/open-telemetry/opentelemetry-js/issues/1096
+ */
+export function getEnvironmentalOrDefaultLogLevel(): LogLevel {
+  return LogLevel.INFO;
```

Comment:
I'm going to work on detecting OTEL_LOG_LEVEL within the browser
( WIP, I'm going to create a suggestion )

Reply:
I think actually it makes sense to merge this first. We may hold release, but you've already done the work here to split the browser/node impls which @iliasbhal will have to do if this isn't merged.

Reply:
ok that works too!